docs/sources/shared/visualizations/axis-options-state-status.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Axis options
+comments: |
+  This file is used in the following visualizations: state timeline, status history.
+---
+
+| Option    | Description                                                                                      |
+| --------- | ------------------------------------------------------------------------------------------------ |
+| Placement | Select the placement of the y-axis. The axis can only be displayed or hidden.                    |
```

Comment:
Sorry @imatwawana , gonna need another round on this. We added a placement option to state timeline and status history but there are only two options - Auto (to display it) and hidden. This is just to provide a subset of the options already available for timeseries. I added back the table, removed the anchor because the placement anchor for these are for the legend placement, and tried to summarize this is a show/hide type situation.

We will be eventually adding in the left/right placement to match what is available for timeseries, but wanting to keep the scope from creeping on this any more than it already has. I realize auto for show is not ideal, but hopefully its functional as a transitional option until we build out the rest.